lib/internal/warn.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// A process Warning is similar in structure to an Error except that they
+// are emitted by the process.on('warn') event rather than being thrown.
+
+const _message = Symbol('message');
```

Comment:
Any specific reason not to use properties for `_name` and `_message`? Is this a common pattern?